### ISSUE TITLE
Bottom Menu

### ISSUE THREAD

**Issue by davidmpurdy** [Mobideck/appdeck#47]
Is there any way to have a bottom menu or "tab bar" - the same style as the Facebook app uses (at least on iOS)?

Thank you!

**Comment by rwanito** [comment]
+1 ;) good idea

**Comment by mdekermadec** [comment]
Yes this is a good idea. Right now we are working on In App purchase implementation. I think that this one should on our todo list :)
